--- micropython-1.14/ports/unix/mpconfigport.orig.mk	2020-10-24 01:16:18.903834937 -0500
+++ micropython-1.14/ports/unix/mpconfigport.mk	2020-10-24 01:16:57.798536852 -0500
@@ -27,14 +27,14 @@
 MICROPY_PY_USSL = 1
 # axTLS has minimal size but implements only a subset of modern TLS
 # functionality, so may have problems with some servers.
-MICROPY_SSL_AXTLS = 1
+MICROPY_SSL_AXTLS = 0
 # mbedTLS is more up to date and complete implementation, but also
 # more bloated.
-MICROPY_SSL_MBEDTLS = 0
+MICROPY_SSL_MBEDTLS = 1
 
 # jni module requires JVM/JNI
 MICROPY_PY_JNI = 0
 
 # Avoid using system libraries, use copies bundled with MicroPython
 # as submodules (currently affects only libffi).
-MICROPY_STANDALONE = 0
+MICROPY_STANDALONE = 0
