Index: uwsgi-2.0.24/core/emperor.c
===================================================================
--- uwsgi-2.0.24.orig/core/emperor.c
+++ uwsgi-2.0.24/core/emperor.c
@@ -730,7 +730,7 @@ void emperor_back_to_ondemand(struct uws
 	c_ui->status = 2;
 	c_ui->cursed_at = uwsgi_now();
 
-	uwsgi_log_verbose("[emperor] bringing back instance %s to on-demand mode\n", c_ui->name);
+	// uwsgi_log_verbose("[emperor] bringing back instance %s to on-demand mode\n", c_ui->name);
 }
 
 void emperor_stop(struct uwsgi_instance *c_ui) {
@@ -760,7 +760,7 @@ void emperor_curse(struct uwsgi_instance
 		c_ui->status = 1;
 	c_ui->cursed_at = uwsgi_now();
 
-	uwsgi_log_verbose("[emperor] curse the uwsgi instance %s (pid: %d)\n", c_ui->name, (int) c_ui->pid);
+	// uwsgi_log_verbose("[emperor] curse the uwsgi instance %s (pid: %d)\n", c_ui->name, (int) c_ui->pid);
 
 }
 
@@ -949,7 +949,7 @@ void emperor_add(struct uwsgi_emperor_sc
 		}
 
 		event_queue_add_fd_read(uwsgi.emperor_queue, n_ui->on_demand_fd);
-		uwsgi_log("[uwsgi-emperor] %s -> \"on demand\" instance detected, waiting for connections on socket \"%s\" ...\n", name, socket_name);
+		// uwsgi_log("[uwsgi-emperor] %s -> \"on demand\" instance detected, waiting for connections on socket \"%s\" ...\n", name, socket_name);
 		return;
 	}
 
@@ -1818,7 +1818,7 @@ void emperor_loop() {
 					if (byte == 17) {
 						ui_current->loyal = 1;
 						ui_current->last_loyal = uwsgi_now();
-						uwsgi_log_verbose("[emperor] vassal %s is now loyal\n", ui_current->name);
+						// uwsgi_log_verbose("[emperor] vassal %s is now loyal\n", ui_current->name);
 						// remove it from the blacklist
 						uwsgi_emperor_blacklist_remove(ui_current->name);
 						// TODO post-start hook
@@ -1846,12 +1846,12 @@ void emperor_loop() {
 					else if (byte == 5) {
 						ui_current->accepting = 1;
 						ui_current->last_accepting = uwsgi_now();
-						uwsgi_log_verbose("[emperor] vassal %s is ready to accept requests\n", ui_current->name);
+						// uwsgi_log_verbose("[emperor] vassal %s is ready to accept requests\n", ui_current->name);
 					}
 					else if (byte == 1) {
 						ui_current->ready = 1;
 						ui_current->last_ready = uwsgi_now();
-						uwsgi_log_verbose("[emperor] vassal %s has been spawned\n", ui_current->name);
+						// uwsgi_log_verbose("[emperor] vassal %s has been spawned\n", ui_current->name);
 					}
 					else if (byte == 2) {
 						emperor_push_config(ui_current);
@@ -1990,7 +1990,7 @@ recheck:
 					ui_current->cursed_at = 0;
 					ui_current->ready = 0;
 					ui_current->accepting = 0;
-					uwsgi_log("[uwsgi-emperor] %s -> back to \"on demand\" mode, waiting for connections on socket \"%s\" ...\n", ui_current->name, ui_current->socket_name);
+					// uwsgi_log("[uwsgi-emperor] %s -> back to \"on demand\" mode, waiting for connections on socket \"%s\" ...\n", ui_current->name, ui_current->socket_name);
 					break;
 				}
 			}
@@ -2323,7 +2323,7 @@ next:
 		uwsgi.has_emperor = 1;
 		uwsgi.emperor_fd = atoi(emperor_env);
 		uwsgi.master_process = 1;
-		uwsgi_log("*** has_emperor mode detected (fd: %d) ***\n", uwsgi.emperor_fd);
+		// uwsgi_log("*** has_emperor mode detected (fd: %d) ***\n", uwsgi.emperor_fd);
 
 		if (getenv("UWSGI_EMPEROR_FD_CONFIG")) {
 			uwsgi.emperor_fd_config = atoi(getenv("UWSGI_EMPEROR_FD_CONFIG"));
